scripts/cluster/agent.py

```diff
@@ -412,25 +413,23 @@ def configure():
     '''
 
     if "service" in configuration:
-        for service in configuration["service"]:
-            print("{}".format(service["name"]))
-            if "arguments_update" in service:
+        for srv in configuration["service"]:
```

Comment:
Why the refactor of this?

Reply:
We called a module `service` so this variable was clashing with it.